### PR TITLE
datashader_cli: add page

### PR DESCRIPTION
**Pull request by wybert**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by wybert** [comment]
@ry feel good for a review

File: pages/common/datashader_cli.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+`datashader_cli points nyc_taxi.parquet --x pickup_x --y pickup_y pickup-scatter.png --background black`
+
+- Visualize the geospaital data, support Geoparquet, shapefile, geojson, geopackage, etc:
+
+`datashader_cli points data.geo.parquet data.png --geo true`
+
+- Use matplotlib to render the image:
+
+`datashader_cli points data.geo.parquet data.png --geo true --matplotlib true`
```
Comment (kbdharun) [inline pages/common/datashader_cli.md:16]: Thanks for your contribution, I have some suggestions for this page.
```suggestion
# datashader_cli

> Quick visualization of large datasets using CLI based on datashader.
> More information: <https://github.com/wybert/datashader-cli>.

- Create a shaded scatter plot of points and save it to a png file, set the background color to black:

`datashader_cli points {{path/to/input.parquet}} --x {{pickup_x}} --y {{pickup_y}} {{path/to/output.png}} --background black`

- Visualize the geospatial data, support Geoparquet, shapefile, geojson, geopackage, etc:

`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo true`

- Use matplotlib to render the image:

`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo {{true}} --matplotlib true`
```

**Review by kbdharun — APPROVED** [review]

File: pages/common/datashader_cli.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# datashader_cli
+
+> Quick visualization of large datasets using CLI based on datashader.
+> More information: <https://github.com/wybert/datashader-cli>.
+
+- Create a shaded scatter plot of points and save it to a png file, set the background color to black:
+
+`datashader_cli points {{path/to/input.parquet}} --x {{pickup_x}} --y {{pickup_y}} {{path/to/output.png}} --background black`
```
Comment (acuteenvy) [inline pages/common/datashader_cli.md:8]:
```suggestion
- Create a shaded scatter plot of points and save it to a png file and set the background color:

`datashader_cli points {{path/to/input.parquet}} --x {{pickup_x}} --y {{pickup_y}} {{path/to/output.png}} --background {{black|white|#rrggbb}}`
```

File: pages/common/datashader_cli.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Visualize the geospatial data, support Geoparquet, shapefile, geojson, geopackage, etc:
+
+`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo true`
```
Comment (acuteenvy) [inline pages/common/datashader_cli.md:12]:
```suggestion
- Visualize the geospatial data (supports Geoparquet, shapefile, geojson, geopackage, etc.):

`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo {{true}}`
```

File: pages/common/datashader_cli.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Use matplotlib to render the image:
+
+`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo {{true}} --matplotlib true`
```
Comment (acuteenvy) [inline pages/common/datashader_cli.md:16]:
```suggestion
`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo {{true}} --matplotlib {{true}}`
```

File: pages/common/datashader_cli.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+> Quick visualization of large datasets using CLI based on datashader.
+> More information: <https://github.com/wybert/datashader-cli>.
+
+- Create a shaded scatter plot of points and save it to a png file and set the background color:
+
+`datashader_cli points {{path/to/input.parquet}} --x {{pickup_x}} --y {{pickup_y}} {{path/to/output.png}} --background {{black|white|#rrggbb}}`
+
+- Visualize the geospatial data (supports Geoparquet, shapefile, geojson, geopackage, etc.):
+
+`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo {{true}}`
```
Comment (kbdharun) [inline pages/common/datashader_cli.md:12]:
```suggestion
`datashader_cli points {{path/to/input_data.geo.parquet}} {{path/to/output_data.png}} --geo true`
```

**Comment by acuteenvy** [comment]
> @acuteenvy Placeholders (`{{}}`) shouldn't be highlighted if only one Boolean argument (true in this case) is a definite requirement for an example. It should be only added if both true and false can be used/passed for the example.

I'm sorry, I saw placeholders around booleans in the next example and I didn't really think about it.

**Review by acuteenvy — APPROVED** [review]

**Comment by kbdharun** [comment]
> I'm sorry, I saw placeholders around booleans in the next example and I didn't really think about it.

No issues 😄 , now that I think about this I haven't added it to the style guide yet. Will add it soon.

**Pull request merged**